sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -1375,9 +1385,9 @@ void initReduLocalAccs(size_t LID, size_t WGSize,
 template <bool UniformPow2WG, typename... LocalAccT, typename... InputAccT,
           typename... ResultT, size_t... Is>
 void initReduLocalAccs(size_t LID, size_t GID, size_t NWorkItems, size_t WGSize,
-                       std::tuple<InputAccT...> LocalAccs,
-                       std::tuple<LocalAccT...> InputAccs,
-                       const std::tuple<ResultT...> Identities,
+                       ReduTupleT<InputAccT...> LocalAccs,
+                       ReduTupleT<LocalAccT...> InputAccs,
+                       ReduTupleT<ResultT...> Identities,
```

Comment:
Same question about `const` here.

Reply:
The tuple is passed by value here, so 'const' does not have any effect, right?

Reply:
Good question, I'm not sure!  I just wanted to check if it was deliberate, because you did more than just change the type.

Reply:
> The tuple is passed by value here, so 'const' does not have any effect, right?

Both versions are equivalent from the caller POV, but `const` protects from modifying the value within the callee scope if needed.